packages/next/server/next-server.ts

```diff
@@ -1380,12 +1383,23 @@ export default class Server {
       ]
     }
 
+    const filesystem = this.getClientFilesystem()
+    const buildManifest = filesystem.promises
+      .readFile(join(this.distDir, BUILD_MANIFEST), 'utf8')
+      .then(JSON.parse)
+
+    const reactLoadableManifest = filesystem.promises
+      .readFile(join(this.distDir, REACT_LOADABLE_MANIFEST), 'utf8')
+      .then(JSON.parse)
```

Comment:
These should be `require`d in production so that they are cached and we aren't reading/parsing them for every page request.